ddtrace/contrib/logging/patch.py

```diff
@@ -59,6 +70,34 @@ def _w_makeRecord(func, instance, args, kwargs):
     return record
 
 
+def _w_StrFormatStyle_format(func, instance, args, kwargs):
+
+    # The format string "dd.service={dd.service}" expects
+    # the record to have a "dd" property which is an object that
+    # has a "service" property
+    # PercentStyle, and StringTemplateStyle both look for
+    # a "dd.service" property on the record
+    try:
```

Comment:
Not a fan of the fail-quiet exception handler - if it fails it should fail loudly imo. Was there a reason behind including it?

Reply:
I was worried about breaking someone's logging, but I suppose this function is actually rather safe... probably overkill having this.

Reply:
removed